app/modules/intelligence/provider/openrouter_gemini_model.py

```
@@ -1,143 +1,380 @@
+            # SDK may strip extra fields; try model_dump() / model_extra for OpenRouter cost
+            if cost is None and hasattr(u, "model_dump"):
+                try:
+                    data = u.model_dump()
+                    cost = data.get("total_cost") or data.get("cost")
+                    if cost is None:
+                        pc = data.get("prompt_cost") or 0
+                        cc = data.get("completion_cost") or 0
+                        if pc or cc:
+                            cost = float(pc) + float(cc)
+                except Exception:
+                    pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)